chain/bitcoind.go

```diff
@@ -359,13 +359,21 @@ func (c *BitcoindClient) Start() error {
 		return errors.New("mismatched networks")
 	}
 
+	// Connect a ZMQ socket for block notifications
+	zmqClient, err := gozmq.Subscribe(c.zmqConnect, []string{"rawblock",
+		"rawtx"}, c.zmqPollInterval)
+	if err != nil {
+		log.Error(err)
```

Comment:
This bit is probably unnecessary since the error is being returned, but it doesn't hurt.